routing.go

```diff
@@ -412,7 +411,7 @@ func (dht *IpfsDHT) getValues(ctx context.Context, key string, stopQuery chan st
 func (dht *IpfsDHT) refreshRTIfNoShortcut(key kb.ID, queries []*query) {
 	shortcutTaken := false
 	for _, q := range queries {
-		if q.localPeers.LenUnqueriedFromKClosest() > 0 {
+		if len(q.queryPeers.GetClosestNotUnreachable(3)) > 0 {
```

Comment:
@aschmahmann 

1. It should be `Beta`, not 3.
2. Wont this always be true ? If we terminate because of `isLookupTermination()`, it will be because this was greater than zero.